definitions.md

```diff
@@ -0,0 +1,66 @@
+# Continuous Delivery Definitions
+
+The purpose of this doc is to record definitions of important terminology in the Continuous
+Delivery space, based on [the canonical definitions of these terms](timeline.md).
+
+This is meant to be a living document: if you would like to propose updates to these definitions,
+[please open a PR](CONTRIBUTING.md).
+
+* [Continuous Delivery](#continuous-delivery)
+* [Continuous Deployoment](#continuous-deployment)
+* [Continuous Integration](#continuous-integration)
+* [CI/CD](#cicd)
+
+## Continuous Delivery
+
+**Continuous Delivery (CD)**: A software development practice in which teams release software changes to users safely,
+quickly and sustainably by:
+
+ 1. Proving that changes can be released at any time
+ 2. Automating release processes
+
+Sources: [Continuous Delivery (2010)](https://www.oreilly.com/library/view/continuous-delivery-reliable/9780321670250/),
+[Martin Fowler](https://www.martinfowler.com/bliki/ContinuousDelivery.html),
+[Jez Humble](https://www.informit.com/articles/article.aspx?p=1833567&seqNum=2)
+
+How does Continuous Delivery relate to:
+* [Continuous Integration](#continuous-integration): Continuous Integration is a practice that makes Continuous Delivery
+  Possible - [Continuous Delivery (2010)](https://www.oreilly.com/library/view/continuous-delivery-reliable/9780321670250/)
+* [Continuous Deployment](#continuous-deployment): Continuous Deployment is made possible by Continuous Delivery -
+  [Jez Humble](https://continuousdelivery.com/2010/08/continuous-delivery-vs-continuous-deployment/),
+  [Timothy Fitz](https://www.blazemeter.com/blog/five-things-you-should-know-about-continuous-deploymentby-man-who-coined-term),
+  [Martin Fowler](https://martinfowler.com/bliki/ContinuousDelivery.html)
+
+## Continuous Deployment
+
+**Continuous Deployment**: Working software is released to users automatically on every commit.
+
+Sources: [Timothy Fitz](http://timothyfitz.com/2009/02/08/continuous-deployment/)
+
+How does Continuous Deployment relate to:
+* [Continuous Delivery](#continuous-delivery): Continuous Deployment is made possible by Continuous Delivery -
+  [Jez Humble](https://continuousdelivery.com/2010/08/continuous-delivery-vs-continuous-deployment/),
+  [Timothy Fitz](https://www.blazemeter.com/blog/five-things-you-should-know-about-continuous-deploymentby-man-who-coined-term),
+  [Martin Fowler](https://martinfowler.com/bliki/ContinuousDelivery.html)
+* [Continuous Integration](#continuous-integration): Continuous Integration is a requirement for Continuous Deployment -
+  [Timothy Fitz](http://timothyfitz.com/2012/11/25/paths-to-continuous-deployment/)
+
+## Continuous Integration
+
+**Continuous Integration (CI)**: The process of combining code changes frequently, where each change (release candidate)
```

Comment:
I'm not sure the "(release candidate)" part is useful here.  There could be legit policy reasons for not having a one to relationship between a commit and a deploy, but that the other elements (automatically validated, etc.) still very much apply.

Reply:
Also, per Terry Cox we should refer to explicit methodologies that can be compared against

Reply:
> Also, per Terry Cox we should refer to explicit methodologies that can be compared against

@tequilarista can you explain a bit more?